main.cpp

```diff
@@ -140,28 +215,29 @@ int main()
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
-    //1)
-    
-    //2)
-    
-    //3)
-    
-    //4)
-    
-    //5)
-    
+    //1) 
+        bachelorsDegree (4, 1);
+    //2) 
+        dayCare (50, 8);
+    //3) 
+        auto goalThisYear = weightLoss (5, 155);
+    //4) 
+        auto internet = subscription (199.99, 3.99); 
+    //5) 
+        auto distanceToSchool = schoolLocation (17, 1); 
     //6)
-    
+        auto blackFriday = salePrice (19.99, 0.5);
     //7)
-    
+        projectManagement (9, 8);
     //8)
-    
+        auto realEstateMarket = rentalProperties (1989, 5367.98);
     //9)
+        auto drumRecording = analogConsole (16, 1);
+    //10)
```

Comment:
This indentation is wrong. 
Code should be indented 4 spaces to the right of the curly braces that surround it
These lines are indented 8 spaces.
- [ ] revise